sketch.js

```diff
@@ -1,18 +1,29 @@
 var amplitude, 
     level,
     song, 
-    speed;
+    speed,
+    amplitudeLevel,
+    count = 0;
 
 function preload(){
   song = loadSound('/assets/FortunateSon.mp3');
 }
 
 function setup() {
-  createCanvas(0, 0)
+  createCanvas(0, 0);
   song.play();
-  peaks = song.getPeaks([width]);
   amplitude = new p5.Amplitude();
-  level = amplitude.getLevel();
+
 }
 
-module.exports = level;
\ No newline at end of file
+function draw() {
+  level = amplitude.getLevel();
+  var size = map(level, 0, 1, 1, 30);
+  count++;
+  if (count === 1) {
+    amplitudeLevel = createElement('span', size).addClass('amplitudeLevel');
+  } else {
+    amplitudeLevel.html('');
+    amplitudeLevel = createElement('span', size).addClass('amplitudeLevel');
```

Comment:
You can use data attributes to keep the amplitude hidden from the Game user.

[deets](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes)